### PR TITLE
Wip new

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
needs :eyes:

@jmburges @AnnJohn

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,62 @@
...
+def using_include(array, element)
+  array.include?(element)
+end
+```
+
+Run your test suite again and you should be passing the first one. 
+
+### `#using_sort`
+
+This method takes in an argument of an array and should return the array, sorted. Use the `#sort` method on that array to get this test passing.
```
Comment (jmburges) [inline README.md]: sorted ascending or descending?

**Comment by jmburges** [comment]
I'm fine with this. Just an idea...do we want to remove the fact that we tell them what methods to use and say hey give this a quick google? we'll effectively give them enough keywords that they should be able to figure it out

**Comment by jmburges** [comment]
(If not, then feel free to :shipit: )

**Comment by SophieDeBenedetto** [comment]
@jmburges going to merge. leaving in the explicit instructions just cuz it's a little confusing to basically write a method that wraps a method. it's contrived so that we can test them.

**Pull request merged**